app-react/src/main/kotlin/com/ustadmobile/view/ReportEditScreen.kt

```diff
@@ -0,0 +1,300 @@
+                            lastName = "Doe"
+                        }
+                    },
+                    ReportFilterWithDisplayDetails().apply {
+                        person = Person().apply {
+                            firstNames = "Ahmad"
+                            lastName = "Ahmadi"
+                        }
+                    }
+                ),
+                deleteButtonVisible = true
+            ),
+        )
+    }
+}
+
+private val ReportEditScreenComponent2 = FC<ReportEditScreenProps> { props ->
+
+    val strings = useStringsXml()
+
+    Container {
+        maxWidth = "lg"
+
+        Stack {
+            direction = responsive(StackDirection.column)
+            spacing = responsive(10.px)
+
+            UstadTextEditField {
+                value = props.uiState.report?.reportTitle ?: ""
+                label = strings[MessageID.xapi_options_report_title]
+                enabled = props.uiState.fieldsEnabled
+                error = props.uiState.titleError
+                onChange = {
+                    props.onReportChanged(
+                        props.uiState.report?.shallowCopyReportWithSeriesWithFilters {
+                            reportTitle = it
+                        })
+                }
+            }
+
+            UstadTextEditField {
+                value = props.uiState.report?.reportDescription ?: ""
+                label = strings[MessageID.description]
+                enabled = props.uiState.fieldsEnabled
+                onChange = {
+                    props.onReportChanged(
+                        props.uiState.report?.shallowCopyReportWithSeriesWithFilters {
+                            reportDescription = it
+                        })
+                }
+            }
+
+            UstadMessageIdDropDownField {
+                value = props.uiState.report?.xAxis ?: 0
+                label = strings[MessageID.xapi_options_x_axes]
+                options = XAxisConstants.X_AXIS_MESSAGE_IDS
+                enabled = props.uiState.fieldsEnabled
+                onChange = {
+                    props.onReportChanged(
+                        props.uiState.report?.shallowCopyReportWithSeriesWithFilters {
+                            xAxis = it?.value ?: 0
+                        })
+                }
+            }
+
+            UstadMessageIdDropDownField {
+                value = props.uiState.report?.reportDateRangeSelection ?: 0
+                label = strings[MessageID.time_range]
+                options = DateRangeConstants.DATE_RANGE_MESSAGE_IDS
+                enabled = props.uiState.fieldsEnabled
+                onChange = {
+                    props.onReportChanged(
+                        props.uiState.report?.shallowCopyReportWithSeriesWithFilters{
+                            reportDateRangeSelection = it?.value ?: 0
+                        })
+                }
+            }
+
+            List{
```

Comment:
List should only be used to contain ListItem. It should be used for the filters and maybe the add series button, but shouldn't contain textfields.